packages/CollapsibleChecklists/src/CollapsibleChecklists.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Heading from "./components/Heading";
+import Group from "./components/Group";
+import Item from "./components/Item";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+const defaultProps = {};
+
+const CollapsibleChecklists = props => {
+  const { children, onChange } = props;
+
+  return (
+    <React.Fragment>
+      {children.map((child, index) => {
+        switch (child.type.displayName) {
```

Comment:
Can we count on the `type` not being changed during transpilation / minification?

Reply:
Yes that identity should be preserved. Especially when there is no hot reloading involved, there should be nothing to worry about AFAIK.

Reply:
I can remove the `.displayName` part

Reply:
I'm kind of hesitant to recommend this though since that would require everyone to use the latest `react-hot-loader`... thoughts?

Reply:
How many apps are using the latest `react-hot-loader` vs the broken one?

Since there are a few components now using this `.displayName` approach, I'm leaning towards keeping it for now, not creating issues in our apps, and capturing this possible refactor in an issue which we can revisit when at least most of our apps have updated their 🔥 loaders.

Reply:
Some of them are not using `react-hot-loader` at all...

Sounds good. Looks like refactoring this will just be a semver-patch.